index.html

```diff
@@ -1,40 +1,45 @@
-<script type="text/javascript">
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-ga('create', 'UA-710527-8', 'schnitmydadsays.com');
-ga('send', 'pageview');
-</script>
-</head>
-<body>
-	<div id="contain">
-		<header>
-			<h1><a href="/">SchnitMyDadSays</a></h1>
-		</header>
-		<div id="main">
-			<img class="avatar" src="assets/images/schnitzel.jpg" style="height:400px;width:400px" alt="Veal Schnitzel">
-			<p class="description">Schnitzel reviews, comments and general chatter. Based out of Sydney, Australia, but good schnit knows no borders. Get into a schnitty!</p>
-			<ul class="links">
-				<li><a href="http://twitter.com/schnitmydadsays">Twitter</a></li>
-				<li><a href="http://facebook.com/schnitmydadsays">Facebook</a></li>
-				<li><a href="https://market.android.com/details?id=com.schnitzel">Android</a></li>
-			</ul>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+		<meta name="google-site-verification" content="c_GZamtpqx8T9vZ0REBwjhg9C-nW1EAfLZ5dD_3ztOw">
+		<meta name="author" content="Brendan Murty">
+		<meta name="description" content="Schnitzel reviews, comments and general chatter.">
+		<meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no">
+		
+		<title>SchnitMyDadSays</title>
+		<link href='http://fonts.googleapis.com/css?family=PT+Serif|Droid+Sans' rel='stylesheet' type='text/css'>
+		<link href="assets/css/main.css?v=20140912a" rel="stylesheet" type="text/css">
+		
+		<script type="text/javascript">
+			(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+			(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+			})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+			ga('create', 'UA-710527-8', 'schnitmydadsays.com');
+			ga('send', 'pageview');
+		</script>
+	</head>
+	<body>
+		<div id="contain">
+			<div id="main">
+				<div class="logo">
+					<span class="schnit">Schnit</span>
+					mydadsays
+					<hr>
+				</div>
+				<ul class="links">
+					<li><a href="http://twitter.com/schnitmydadsays">Twitter</a></li>
+					<li><a href="http://facebook.com/schnitmydadsays">Facebook</a></li>
+					<li><a href="https://market.android.com/details?id=com.schnitzel">Android</a></li>
+				</ul>
+			</div>
+			<footer>
+				<p>Web development by <a href="http://brendanmurty.com/">Brendan Murty</a> and <a href="https://github.com/brendanmurty/schnitmydadsays">shared on GitHub</a>.</p>
+				<p>Delicious background image courtesy of <a href="http://www.bonappetit.com/" target="_blank">Bonappetit.com</a>
```

Comment:
Not fussed either way. We should try and get some better shots of Schnits ourselves potentially. Then again, most Schnits we have are likely at night and/or in low light conditions.